src/screens/PinCodeUnlock/PinCodeUnlock.js

```diff
@@ -198,9 +222,12 @@ class PinCodeUnlock extends React.Component<Props, State> {
     }
   };
 
-  handlePinSubmit = (pin: string) => {
+  handlePinSubmit = async (pin: string) => {
```

Comment:
```suggestion
  handlePinSubmit = (pin: string) => {
```

Reply:
It should be async now - waiting on this.handleUnlockAction to be finished.